client/blocks/edit-gravatar/index.jsx

```diff
@@ -51,12 +112,40 @@ export class EditGravatar extends Component {
 		// send gravatar request
 		if ( bearerToken ) {
 			debug( 'Got the bearerToken, sending request' );
-			uploadGravatarAction( files[ 0 ], bearerToken, user.email );
+			uploadGravatarAction( imageBlob, bearerToken, user.email );
 		} else {
 			debug( 'Oops - no bearer token.' );
 		}
 	}
 
+	hideImageEditor = ( imageEditorProps ) => {
+		const { resetAllImageEditorState } = imageEditorProps;
```

Comment:
Just a heads up that we'll likely refactor this in the future so callers need to import the `resetAllImageEditorState` action instead of being passed back in the callback